_layouts/default.html

```diff
@@ -270,45 +270,45 @@ <h1 class="project-name">{{ page.title | default: site.title | default: site.git
         <div id="myDropdownMlops" class="dropdown-mlops">
-            <a href="http://localhost:4000/mlops_summary.html">Summary</a>
-            <a href="http://localhost:4000/requirements_research.html">Requirements and Research</a>
-            <a href="http://localhost:4000/versioning.html">Data Versioning</a>
-            <a href="http://localhost:4000/DAG.html">Training Pipeline</a>
-            <a href="http://localhost:4000/serving.html">Model Serving</a>
-            <a href="http://localhost:4000/pred_service.html">Prediction Service</a>
-            <a href="http://localhost:4000/monitoring.html">Model Monitoring</a>
-            <a href="http://localhost:4000/gitops.html">GitOps</a>
-            <a href="http://localhost:4000/team_arch.html">Team Architecture</a>
+            <a href="http://localhost:4000/mlops_big_picture/mlops_summary.html">Summary</a>
+            <a href="http://localhost:4000/mlops_big_picture/requirements_research.html">Requirements and Research</a>
+            <a href="http://localhost:4000/mlops_big_picture/versioning.html">Data Versioning</a>
+            <a href="http://localhost:4000/mlops_big_picture/DAG.html">Training Pipeline</a>
+            <a href="http://localhost:4000/mlops_big_picture/serving.html">Model Serving</a>
+            <a href="http://localhost:4000/mlops_big_picture/pred_service.html">Prediction Service</a>
+            <a href="http://localhost:4000/mlops_big_picture/monitoring.html">Model Monitoring</a>
+            <a href="http://localhost:4000/mlops_big_picture/gitops.html">GitOps</a>
+            <a href="http://localhost:4000/mlops_big_picture/team_arch.html">Team Architecture</a>
         </div>
         <a onclick="toggleNavBusiness()"><img src="https://cdn0.iconfinder.com/data/icons/rounded-basics/24/rounded__menu-512.png" width="15px" height="15px"> Corporate Perspective</a>
         <div id="myDropdownBusiness" class="dropdown-business">
-            <a href="http://localhost:4000/deployment_lifecycle.html">Deployment Service Life Cycle</a>
-            <a href="http://localhost:4000/prerequisites.html">Skills, Roles and Tool Horizon Scan</a>
-            <a href="http://localhost:4000/maturity_assessment.html">Maturity Assessment</a>
-            <a href="http://localhost:4000/best_practices.html">MLOps Best Practices</a>
+            <a href="http://localhost:4000/corporate_perspective/deployment_lifecycle.html">Deployment Service Life Cycle</a>
+            <a href="http://localhost:4000/corporate_perspective/prerequisites.html">Skills, Roles and Tool Horizon Scan</a>
+            <a href="http://localhost:4000/corporate_perspective/maturity_assessment.html">Maturity Assessment</a>
+            <a href="http://localhost:4000/corporate_perspective/best_practices.html">MLOps Best Practices</a>
```

Comment:
Can we make these relative URLs instead of absolute?  That way it should work for both local and github based links?